applications/obsloctap/templates/ingress.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: gafaelfawr.lsst.io/v1alpha1
+kind: GafaelfawrIngress
+metadata:
+  name: "obsloctap"
+  labels:
+    {{- include "obsloctap.labels" . | nindent 4 }}
+config:
+  baseUrl: {{ .Values.global.baseUrl | quote }}
+  scopes:
+    all:
+      - "exec:portal"
```

Comment:
```suggestion
      - "read:tap"
```

Reply:
should this be read:all or read:any

Reply:
I do probably want a web page as well as an API